### PR TITLE
Add PIS providers for sparda, dab & consors

### PR DESCRIPTION
**Pull request by artemsnisarenko**

### CODE REVIEW

**Comment by artemsnisarenko** [comment]
PIS/PaymentInit/Products/001a & PIS/GetTransactionStatus/001a pass for all three.

File: adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java [hunk 1]
```diff
@@ -26,8 +26,9 @@
 
 public class AdorsysIntegAccountInformationServiceProvider implements AccountInformationServiceProvider {
```
Comment (n0rdy) [inline adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java]: Could you, please, rename this and the rest providers to the name convention, you used for Sparda/DAB/Consors? AdorsysIntegAccountInformationServiceProvider -> AdorsysIntegServiceProvider.
Thanks!

Comment (artemsnisarenko) [reply adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java]: AdorsysIntegAccountInformationServiceProvider provides only one service. I don't think it should be renamed.

Comment (n0rdy) [reply adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java]: According to the providers' design, that was introduced by current PR, such services as `AdapterNameAccountInformationServiceProvider` and `AdapterNamePaymentInitiationServiceProvider` don't exist no more. From now on there is only `AdapterNameServiceProvider`, that provides some number of services. 
That's why, I think, that we should follow the one convention in our project, but not support both (the old and the new ones).

Comment (artemsnisarenko) [reply adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java]: Merged and renamed rest of the providers.

Comment (n0rdy) [reply adapters/adorsys-integ-adapter/src/main/java/de/adorsys/xs2a/gateway/adorsys/service/provider/AdorsysIntegAccountInformationServiceProvider.java]: Thanks!

**Pull request merged**